app/views/search/_results_for_pre_create_petition.html.erb

```diff
@@ -1,12 +1,17 @@
 <% petitions.each do |petition| %>
   <ul>
-    <li>
-      <%= link_to petition.title, petition_path(petition), class: 'link_prompt text_link' %>
+    <li class="petition-item petition-item-existing">
+      <div class="panel-indent">
+        <%= link_to petition.title, petition_path(petition) %>
+        <p>
+          <% unless petition.rejected? -%>
+            <%= number_with_delimiter(petition.signature_count) %> signatures
```

Comment:
If you rebase against master you can get the stuff @pixeltrix did in #134 to ward against "1 signatures"